src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/Ink/LassoSelectionBehavior.cs

```diff
@@ -426,16 +427,14 @@ private static ElementCornerPoints GetTransformedElementCornerPoints(InkCanvasIn
         /// Private helper that will generate a grid of points 5 px apart given the elements bounding points
         /// this works with any affline transformed points
         /// </summary>
-        private Point[] GeneratePointGrid(ElementCornerPoints elementPoints)
+        private Span<Point> GeneratePointGrid(ElementCornerPoints elementPoints)
```

Comment:
Could you return `ReadOnlySpan<Point>` instead of `Span<Point>` ? There is no perf difference and it just makes it clear that it shouldn't be mutated.

Reply:
As per other discussion and original description, ROS it is.